src/app/shared/constants/constants.ts

```diff
@@ -17,7 +17,7 @@ export class Constants {
   static readonly MAX_TIME = '23:59';
   static readonly MAX_DESCRIPTION_LENGTH = 500;
   static readonly MAX_TEACHER_DESCRIPTION_LENGTH = 300;
-  static readonly PHONE_LENGTH = 10;
```

Comment:
I ve already seen same chages in another PR could you please validate if they are still needed?